plugins/GSdx/GSRenderer.cpp

```diff
@@ -242,6 +242,12 @@ bool GSRenderer::Merge(int field)
 
 		fs.x = max(fs.x, (int)(dst[i].z + 0.5f));
 		fs.y = max(fs.y, (int)(dst[i].w + 0.5f));
+		// Fixing Time Crisis 2, 3 split screen view. Without this only the 1p screen is visible.
++		if(src[i].right - src[i].left == 0.5) {
++			src[i].left = 0;
++			src[i].right = 1;
++			dst[i].top += dst[i].bottom*0.25;
++			dst[i].bottom -= dst[i].bottom*0.25;
 	}
```

Comment:
you miss a ``}`` over here. also remove all the ``+`` from the code. to make it atleast compile on Appveyor so users could test whether it works :p